src/stores/SchemaStore.ts

```diff
@@ -257,7 +258,7 @@ export class SchemaStore {
 			await this.selectSchema(schema);
 			this.selectEntityFromURLParams();
 		} else if (this.schemas.length > 0) {
-			this.selectSchema(this.schemas[0]);
+			this.selectSchema(this.schemas[1]);
```

Comment:
this is temporary, because first schema test16rep doesn't work with new structure and props names